src/Organizer/EditOrganizerRestController.php

```diff
@@ -157,13 +157,34 @@ public function updateName(
     }
 
     /**
+     * @deprecated Use updateAddress with language parameter instead.
+     *
      * @param string $organizerId
      * @param Request $request
      * @return JsonResponse
+     *
      * @throws DataValidationException
      */
-    public function updateAddress($organizerId, Request $request)
-    {
+    public function updateAddressDeprecated(
+        string $organizerId,
+        Request $request
+    ) {
+        return $this->updateAddress($organizerId, 'nl', $request);
```

Comment:
Is deze echt helemaal backwards compatible, met de hardcoded 'nl' twijfel ik daar wat aan? Voordien werd op deze plaats in de code geen taal gebruikt, dus zou je nu ook verwachten dat een adres dat je aan deze endpoint doorspeelt steeds voor de hoofd-taal van de organizer is. Wat gebeurt er als je een organizer in het Frans aanmaakt (OrganizerCreatedWithUniqueWebsite event heeft een hoofdtaal-parameter), en dan hier naar het deprecated endpoint een adres doorspeelt?

Indien mijn bedenking hierover terecht is, is het misschien beter om het deprecated endpoint gewoon weg te halen. Of om in dit geval geen taal in het commando te steken, en pas in de entiteit zelf bij gebrek aan taal dit als adres voor de hoofdtaal te gebruiken.

Reply:
Ik heb dit endpoint erin gelaten om te zorgen dat de angular app blijft werken zonder aanpassing. De angular app maakt steeds een nieuwe organizer met nl als main language, dus als daarna een update van het adres wordt doorgevoerd met het oude endpoint is dat ook steeds met nl. Als de angular app is aangepast mogen/moeten deze deprecated endpoints asap verdwijnen.

Reply:
Zie voor opvolging https://jira.uitdatabank.be/browse/III-2785